### DISCUSSION TITLE
Studio Install error

### DISCUSSION THREAD

**Discussion by Luukveltkamp** [tinymovr/Tinymovr#335] [General]
Hi, I am very new to this and have reached an roadblock while installing Studio using pip.

WARNING: typer 0.12.3 does not provide the extra 'all'

Error occurs with both GUI and CLI installations .

I am using Python 3.12.3

**Comment by yconst** [comment]
It seems this issue is due to a broken dependency -- `typer`. As a temp solution, you can install an older version of typer:
```
pip uninstall typer
pip install typer==0.11
```

you might need to install some additional dependencies, in my case I had to install `colorama`.
`typer` is an indirect dep of our Avlos library, but not an essential one. We'll publish an update that makes this optional.
